packages/core/src/payment/payment-strategy-action-creator.ts

```diff
@@ -19,6 +21,7 @@ import { PaymentStrategy } from './strategies';
 export default class PaymentStrategyActionCreator {
     constructor(
         private _strategyRegistry: PaymentStrategyRegistry,
+        private _strategyRegistryV2: ResolveIdRegistry<PaymentStrategyV2, PaymentStrategyResolveId>,
```

Comment:
You could import `PaymentStrategyRegistry` from `payment-strategy-registry-v2`, which is essentially an alias for `ResolveIdRegistry<PaymentStrategyV2, PaymentStrategyResolveId>`.